README.md

```diff
@@ -33,7 +42,7 @@ python3 -m pip install taichi
 
 **Supported OS**: Windows, Linux, Mac OS X; **Python**: 3.6/3.7/3.8 (64-bit only); **Backends**: x64 CPUs, CUDA, Apple Metal, OpenGL Compute Shaders.
 
-Please build from source for other configurations (e.g., your CPU is ARM).
+Please build from source for other configurations (e.g., your CPU is ARM, or you want to try out our experimental C backend).
 
 **Note:**
  - Starting April 13 2020 (v0.5.12), we release the Python package [`taichi`](https://pypi.org/project/taichi/) instead of [`taichi-nightly`](https://pypi.org/project/taichi-nightly/). Now this PyPI package includes CPU, CUDA 10/11, Metal and OpenGL support.
```

Comment:
```suggestion
 - Starting April 13 2020 (v0.5.12), we release the Python package [`taichi`](https://pypi.org/project/taichi/) instead of [`taichi-nightly`](https://pypi.org/project/taichi-nightly/). Now this PyPI package includes CPU, CUDA 10/11, Metal, and OpenGL support.
```

just a comma between listing objects. :>

Reply:
I thought we don't need `,` when there's `and` following after? Not super clear about English..

Reply:
I do remember it's needed the last independent items in a list (longer than 2 items) from my technical writing courses. But some online software also says this is optional :+1:

Reply:
OIC! Thank for the professional answer!